example/mini_core.rs

```diff
@@ -1,6 +1,14 @@
 #![feature(
-    no_core, lang_items, intrinsics, unboxed_closures, extern_types,
-    decl_macro, rustc_attrs, transparent_unions, auto_traits, freeze_impls,
+    no_core,
```

Comment:
In the future, please put the formatting changes in a different commit to make the review easier.